python/determineLuminosity.py

```diff
@@ -354,11 +360,11 @@ def simulateDataOnHimster(
                     # why are these newly assigned? Have they changed?
                     rec_par.use_xy_cut = thisScenario.use_xy_cut
                     rec_par.use_m_cut = thisScenario.use_m_cut
-                    rec_par.reco_ip_offset = [
-                        ip_info_dict["ip_offset_x"],
-                        ip_info_dict["ip_offset_y"],
-                        ip_info_dict["ip_offset_z"],
-                    ]
+                    #rec_par.reco_ip_offset = [
+                    #    ip_info_dict["ip_offset_x"],
+                    #    ip_info_dict["ip_offset_y"],
+                    #    ip_info_dict["ip_offset_z"],
+                    #]
```

Comment:
The reci ip dict is changed [here](https://github.com/panda-luminosity-detector-group/LuminosityFit/blob/cad5c87b0cb87ce177cbe42d8b2bf52ae72b7a4b/python/determineLuminosity.py#L209-L212) after a first reconstruction run. At this point, the IP may have changed but that's not known to the experimentConfig yet. Maybe we should write that to a log somewhere. I would prefer not to change the config file itself, because that is supposed to be the known start config.

Reply:
Same as previous comment then?

Reply:
Yes!